bytecomp/meta.mli

```diff
@@ -20,14 +20,8 @@ external realloc_global_data : int -> unit = "caml_realloc_global"
 type closure = unit -> Obj.t
 type bytecode
 external reify_bytecode :
-<<<<<<< HEAD
-  bytes array -> Debug_event.debug_event list array -> string option ->
-||||||| 121bedcfd2
-  bytes array -> Instruct.debug_event list array -> string option ->
-=======
   (char, Bigarray.int8_unsigned_elt, Bigarray.c_layout) Bigarray.Array1.t ->
-  Instruct.debug_event list array -> string option ->
->>>>>>> 5.2.0
+  Debug_event.debug_event list array -> string option ->
```

Comment:
These are the same type, no? Could we use `Instruct.debug_event` here just to keep the diff down?

Reply:
I think it's deliberately `Debug_event` to avoid a circular dependency, or similar.  I can't quite remember the details but would rather leave this as-is.